shoop_tests/core/test_basic_order.py

```diff
@@ -17,8 +18,8 @@
 
 
 def create_order(request, creator, customer, product):
-    billing_address = get_address()
-    shipping_address = get_address(name="Shippy Doge")
+    billing_address = get_address(immutable=True)
+    shipping_address = get_address(immutable=True, name="Shippy Doge")
```

Comment:
Same here: `get_address().immutable()` and `get_address(name="Shippy Doge").immutable()` would do